scripts/Diffraction/isis_powder/abstract_inst.py

```diff
@@ -67,7 +67,8 @@ def _focus(self,
                run_number_string,
                do_van_normalisation,
                do_absorb_corrections,
-               sample_details=None):
+               sample_details=None,
+               absorb_method=None):
```

Comment:
Not sure you need this absorb_method parameter now that you've removed it from the onward call to focus